src/routes/scheduledItems.integration.ts

```diff
@@ -0,0 +1,95 @@
+import { expect } from 'chai';
```

Comment:
existing test, didnt change any assertions/logics.

only change is mock `stories.getStories` as the router and getstories were in same file and mock didnt work

and changed the `requestAgent` to call `request(app).get`